src/test/java/org/jitsi/rtp/rtcp/rtcpfb/transport_layer_fb/tcc/RtcpFbTccPacketTest.kt

```diff
@@ -32,30 +32,34 @@ class RtcpFbTccPacketTest : ShouldSpec() {
         0x32, 0x0f, 0x22, 0x3a,
         // Media source SSRC = 2397376430
         0x8e, 0xe5, 0x0f, 0xae,
-        // Base seq num = 1969, packet status count = 9
-        0x07, 0xb1, 0x00, 0x09,
+        // Base seq num = 0xfffa, packet status count = 9
```

Comment:
why did the test values here need to change?

Reply:
To make sure the case when a wrap occurs is tested.

Reply:
To make sure the case when a wrap occurs is tested.

Reply:
i see, but why the change to hex and removing the comment which details the delta values?   i don't think the rollover is that much less obvious with decimal and i find it much easier to parse when reading the test (plus also the delta values and the comment about what the  serialized delta values are?

Reply:
I changed it because I find the exact opposite. The values in microseconds (54, etc) are not needed here and only add an unnecessary layer of translation that one has to do. The comment is removed because that whole layer is not necessary anymore.

When I was reading the code, I wanted to see if I need to update the values in the map, so I wanted to asnwer the question: why is the first value in the map 54? It's because `54.toTicks()` (I had to follow `toTicks` to see what it does) equals 216 which is 0xd8, which is the first entry in the map.

With this change the first value in the map is just equal to the first delta (as the comment describes).



>  i don't think the rollover is that much less obvious with decimal

I agree that the rollover is not harder to see in decimal. But if we consistently one system, we don't have to convert between the two.. And there are good reasons use hex for the byte values. So the value in the `ByteArray` definition is going be 0xfffa. It takes less effort to use it as it is below than to convert it to decimal.